app/controllers/provider_foreman_controller.rb

```diff
@@ -619,6 +675,28 @@ def miq_search_node
     @right_cell_text = _("All %{title} Configured Systems") % {:title => ui_lookup(:ui_title => "foreman")}
   end
 
+  def cm_job_templates_list(id, model)
+    return cm_job_template_node(id, model) if id
+    @listicon = "cm_job_template"
+    if x_active_tree == :cm_job_templates_tree
+      options = {:model => model.to_s}
+      @right_cell_text = _("All Ansible Job Templates")
```

Comment:
Should the text be so Ansible specific?  If so, it would be more accurate to say "All Ansible Tower Job Templates".  The method appears to be generic enough for reuse other than this string.